### DISCUSSION TITLE
Different language GuardRails

### DISCUSSION THREAD

**Discussion by SaroLovito** [NVIDIA-NeMo/Guardrails#266] [Q&A]
I want to implement guardrails for three different languages: 
What do you think is the clearest, most scalable, and maintainable way to do this? Create 3 different rails.co (e.g. english_rails.co, spanish_rails.co, italian_rails.co)?

**Comment by drazvan** [comment]
@SaroLovito: are you planning to deploy a multi-lingual guardrails configuration? or individual ones for each language?

**Reply by SaroLovito** [reply]
My idea is to deploy a multi-lingual guardrails configuration.

**Comment by d-shree** [comment]
@drazvan - Is there any update on this? I have a similar requirement.

**Reply by drazvan** [reply]
Unfortunately, no update on this at the moment.

**Reply by d-shree** [reply]
Thanks for the update!
